packages/migrate/src/glint-utils.ts

```diff
@@ -106,20 +104,20 @@ export async function createGlintService(basePath: string): Promise<GlintService
   return new GlintService(glintCore, basePath);
 }
 
-export async function createGlintFixPlugin(): Promise<GlintFixPlugin> {
+export async function getGlintFixPlugin(): Promise<GlintFixPluginCtor> {
```

Comment:
Just for clarification, changing this from `create` to `get` is because we're not instantiating something.